pandas/tests/groupby/test_function.py

```diff
@@ -76,6 +76,34 @@ def test_groupby_bool_aggs(agg_func, skipna, vals):
     tm.assert_frame_equal(result, exp_df)
 
 
+@pytest.mark.parametrize("bool_agg_func", ["any", "all"])
+def test_bool_aggs_dup_column_labels(bool_agg_func):
+    # 21668
+    df = DataFrame([[True, True]], columns=["a", "a"])
```

Comment:
can you add in skipna as a parameter in these

Reply:
Have added skipna parameterization to `test_bool_aggs_dup_column_labels` (this was just a test moved to keep any/all in same place). 

Would prefer to hold off on adding `skipna` parameterization to the nullable test below because the `skipna=False` behavior currently doesn't follow Kleene logic so current output won't be correct (there's #37506 open for this)

Reply:
Have added skipna parameterization to `test_bool_aggs_dup_column_labels` (this was just a test moved to keep any/all tests together)

For the added nullable dtype test below, problem is that `skipna=False` case doesn't currently give the correct output (#37506)